cpp-inference/inference_scaling_imagenet.cpp

```diff
@@ -2,6 +2,47 @@
 #include "client_test_utils.h"
 #include <mpi.h>
 
+
+
+// some helpers for handling model settings from the
+// driver script in colocated and non-colocated cases
+int get_batch_size() {
+  char* batch_setting = std::getenv("SS_BATCH_SIZE");
+  int batch_size = batch_setting ? std::stoi(batch_setting) : 1;
+  return batch_size;
+}
+
+std::string get_device() {
+  char* device = std::getenv("SS_DEVICE");
+  std::string device_str = device ? device : "GPU";
+  return device_str;
+}
+
+int get_num_devices() {
+  char* num_dev_setting = std::getenv("SS_NUM_DEVICES");
+  int num_devices = num_dev_setting ? std::stoi(num_dev_setting) : 1;
+  return num_devices;
+}
+
+bool get_set_flag() {
+  char* set_flag = std::getenv("SS_SET_MODEL");
+  bool should_set = set_flag ? std::stoi(set_flag) : false;
+  return should_set;
+}
+
+bool get_cluster_flag() {
+  char* cluster_flag = std::getenv("SS_CLUSTER");
+  bool use_cluster = cluster_flag ? std::stoi(cluster_flag) : false;
+  return use_cluster;
+}
+
+int get_client_count() {
+  char* client_count_flag = std::getenv("SS_CLIENT_COUNT");
+  int client_count = client_count_flag ? std::stoi(client_count_flag) : 18;
+  return client_count;
```

Comment:
Recommend making a helper routine:

```
int get_env_integer(const char *var_name, int default)
{
   char *var_str = std::getenv(var_name);
   return (var_str == NULL) ? default : std::stoi(var_str);
}
```
With that, get_client_count (for example) becomes:

```
int get_client_count() {
  return get_env_integer("SS_CLIENT_COUNT", 18);
}
```

Reply:
Also, 18 is a weird default -- please add a comment explaining where that comes from